store/db/mysql/mysql.go

```diff
@@ -67,6 +67,11 @@ func (*DB) BackupTo(context.Context, string) error {
 	return errors.New("Please use mysqldump to backup")
 }
 
+func (*DB) GetCurrentDBSize(context.Context) (int64, error) {
+	// return 0, errors.New("Not implemented")
```

Comment:
```suggestion
	// TODO: implement this.
```

Reply:
Done.
I have a question about this feature. If I modified my setting to Local file. but the memos message was still in Database. In this scenario, what's the "Database file size" means?  database size or object storage size?

Reply:
It's the size of database file. Not the image resources in local file system.